### PR TITLE
Active STN integration branch

### PR DESCRIPTION
**Pull request by msheiny**
Leaving this open for up-to-date peer review and collaboration.

### CODE REVIEW

File: defaults/main.yml [hunk 1]
```diff
@@ -11,6 +12,7 @@ django_stack_gcorn_port: 8000
 django_stack_gcorn_log: /var/log/{{django_stack_app_name}}/app
 django_stack_gcorn_app: freedompress.wsgi
 django_stack_gcorn_app_settings: freedompress.settings.production
```
Comment (conorsch) [inline defaults/main.yml]: Seems like these vars would be better structured as `{{ django_stack_app_name }}.wsgi` and `{{ django_stack_app_name }}.settings.production`. Might still require overrides, but makes more sense to infer namespace.

Comment (msheiny) [reply defaults/main.yml]: Something like this is a good compromise?

``` yaml
 django_stack_gcorn_app: "{{ global_app_name }}.wsgi"
 django_stack_gcorn_app_settings: "{{ global_app_name }}.settings.production"
```

Comment (msheiny) [reply defaults/main.yml]: oh wait

Comment (msheiny) [reply defaults/main.yml]: hahaha

Comment (conorsch) [reply defaults/main.yml]: Sure, as long as the "global" var is also namedspace, e.g. `{{ django_stack_global_app_name }}`.

Comment (msheiny) [reply defaults/main.yml]: Okay I added what you pointed out in latest commit. :+1: Good looking out buddy

**Pull request merged**